ghost/core/core/server/services/content-gating/gated-blocks.js

```
@@ -0,0 +1,114 @@
+    const params = {};
+
+    const matches = paramsString.matchAll(GATED_BLOCK_PARAM_REGEX);
+    for (const match of matches) {
+        const key = match.groups.key;
+        const value = parseStringWithBooleans(match.groups.value);
+
+        if (!ALLOWED_GATED_BLOCK_KEYS.includes(key)) {
+            continue;
+        }
+
+        if (typeof value !== ALLOWED_GATED_BLOCK_PARAMS[key].type) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)